scripts/build/create_commands.py

```diff
@@ -59,7 +61,14 @@ def copy_entry_points(input_data, build_dir):
 
                     LOG.info("Registering sub-command '%s' installed to "
                              "PATH", command_name)
-                shutil.copy2(input_file, package_bin)
+                # Only the CodeChecker top level script should be copied
+                # into the bin directory and every subcommand should be
+                # called through that.
+                if not input_file.startswith("codechecker-"):
```

Comment:
It will also move the `CodeChecker` script to this directory.

By the way what do you think to move these commands into the `package_root/cc_bin` directory?

Reply:
I thought about it but I think we should discuss what should go where.
Right now the cc_bin contains the external binaries (clang ..) if they are copied into the package and the `CodeChecker.py` and `commands.json` files.
Should we keep the cc_bin for only the external binaries if yes where should we put the `CodeChecker.py` and `commands.json` files.  CodeChecker.py needs to be executable because it is called from the CodeChecker top executable through Popen, so it can not be moved to the `bin` directory I think that is why it is in the cc_bin dir.
It would be nice to collect all the codechececker commands related files at one place and close to each other.